tests/utils/sampler_test.py

```diff
@@ -128,28 +126,7 @@ def test_dirichlet_sampler_without_hash_key() -> None:
     sampler_2 = DirichletLabelBasedSampler(unique_labels=list(range(10)), sample_percentage=1.0, beta=0.1)
 
     # Training
-    ds = MnistDataset(data_path=Path("examples/datasets/MNIST"), train=True)
-    samples_per_class = [ds.targets[ds.targets == i].size(0) for i in range(10)]
-
-    new_ds_1 = sampler_1.subsample(ds)
-    new_samples_per_class_1 = [new_ds_1.targets[new_ds_1.targets == i].size(0) for i in range(10)]
-
-    new_ds_2 = sampler_2.subsample(ds)
-    new_samples_per_class_2 = [new_ds_2.targets[new_ds_2.targets == i].size(0) for i in range(10)]
-
-    _, p_val_1 = chisquare(f_obs=new_samples_per_class_1, f_exp=samples_per_class)
-    _, p_val_2 = chisquare(f_obs=new_samples_per_class_2, f_exp=samples_per_class)
-    _, p_val_3 = chisquare(f_obs=new_samples_per_class_1, f_exp=new_samples_per_class_2)
-
-    # Assert that the new distribution with sampler_1 is different from the original distribution
-    assert p_val_1 < 0.01
-    # Assert that the new distribution with sampler_2 is different from the original distribution
-    assert p_val_2 < 0.01
-    # Assert that the new distributions with sampler_1 and sampler_2 are different due to different random seeds
-    assert p_val_3 < 0.01
-
-    # Testing
-    ds = MnistDataset(data_path=Path("examples/datasets/MNIST"), train=False)
+    ds = SYNTHETIC_DATASET
```

Comment:
Same as above, we were essentially running the same test for the two MNIST splits. So I just removed the redundancy.